### PR TITLE
DCS-1786 Hiding images if there isn't a single match

### PR DESCRIPTION
**Pull request by andrewrlee**
Also clearing out alt text if we show the placeholder after talking to accessibility

If we are showing the placeholder as we have a single match but no photo - we can't easily detect this as its coming directly from prison-api. In this case we will continue to show the placeholder

### CODE REVIEW

File: integration_tests/integration/bookedtoday/arrivals/autoMatchingRecords/hasMultipleMatches.cy.ts [hunk 1]
```diff
@@ -73,7 +73,10 @@ context('Arrival matches multiple records', () => {
       match.fieldName('prison-number').should('contain', arrival.potentialMatches[0].prisonNumber)
       match.fieldName('pnc-number').should('contain', arrival.potentialMatches[0].pncNumber)
       match.fieldName('cro-number').should('contain', arrival.potentialMatches[0].croNumber)
-      match.prisonerImage().should('have.attr', 'src', `/prisoners/${arrival.potentialMatches[0].prisonNumber}/image`)
+      match.prisonerImage.check({
```
Comment (andrewrlee) [inline integration_tests/integration/bookedtoday/arrivals/autoMatchingRecords/hasMultipleMatches.cy.ts:76]: Most of the changes here are moving to use a new utility to test image attrributes

File: integration_tests/integration/bookedtoday/choosePrisoner.cy.ts [hunk 2]
```diff
@@ -75,39 +75,44 @@ context('Choose Prisoner', () => {
...
+      isCurrentPrisoner: false,
+      potentialMatches: [],
+    })
+    cy.task('stubExpectedArrivals', { caseLoadId: 'MDI', arrivals: [arrival] })
+
+    cy.signIn()
+    const choosePrisonerPage = ChoosePrisonerPage.goTo()
+    choosePrisonerPage.prisonerImage(0).check({
+      href: '/assets/images/placeholder-image.png',
+      alt: '',
```
Comment (andrewrlee) [inline integration_tests/integration/bookedtoday/choosePrisoner.cy.ts:94]: when placeholder shown there is no alt text

File: integration_tests/pages/page.ts [hunk 3]
```diff
@@ -17,6 +17,17 @@ export default abstract class Page {
...
+  static checkImage(element: PageElement, href: string, alt: string) {
+    element.should('be.visible').then(e => {
+      cy.wrap(e)
+        .should('have.attr', 'src')
+        .then(src => expect(src).equal(href))
+      cy.wrap(e)
+        .should('have.attr', 'alt')
+        .then(altAttr => expect(altAttr).equal(alt))
+    })
+  }
```
Comment (andrewrlee) [inline integration_tests/pages/page.ts:29]: shared code to check image attributes

File: server/views/partials/expectedArrivalCardMacro.njk [hunk 4]
```diff
@@ -50,7 +50,9 @@
     <div class='app-card__content'>
         <div class="app-card app-card--with-aside">
             <aside class="app-card__aside">
-                <img src="{{ '/prisoners/' + arrival.potentialMatches[0].prisonNumber + '/image' if arrival.potentialMatches[0].prisonNumber else '/assets/images/placeholder-image.png' }}" class="app-card__image" alt="{{ arrival.lastName }}, {{ arrival.firstName }}" data-qa="prisoner-image"/>
+                <img 
+                  src="{{ '/prisoners/' + arrival.potentialMatches[0].prisonNumber + '/image' if arrival.matchType === 'SINGLE_MATCH' else '/assets/images/placeholder-image.png' }}" class="app-card__image" 
+                  alt="{{ arrival.lastName + ', ' + arrival.firstName if arrival.matchType === 'SINGLE_MATCH' else '' }}" data-qa="prisoner-image"/>
             </aside>
```
Comment (andrewrlee) [inline server/views/partials/expectedArrivalCardMacro.njk:56]: the actual change - only show image if a single match

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Review by michalnawrockidj — APPROVED** [review]
LGTM

**Pull request merged**